frontend/compositions/visualizations/map/useSearchData.tsx

```diff
@@ -0,0 +1,31 @@
+import { geoAlbersUsa } from "d3"
+import { Feature, FeatureCollection } from "geojson"
+import { useSearch } from "../../../helpers"
+import { Incident } from "../../../helpers/api"
+
+export default function useSearchData() {
+  const { incidentResults } = useSearch()
+  const projection = geoAlbersUsa()
+    .scale(1300)
+    .translate([487.5 + 112, 305 + 50])
```

Comment:
could we possibly use variable names instead of magic numbers?

Reply:
These numbers are the projection settings that come in the map boundary data, so to me it's clearer/more error-proof not to make variables.  ```projection``` will be a shareable variable soon and will wrap these.

Reply:
These numbers are the projection settings that come in the map boundary data, so to me it's clearer/more error-proof not to make variables.  ```projection``` will be a shareable variable soon and will wrap these.